src/measure_theory/measurable_space.lean

```diff
@@ -95,6 +95,14 @@ protected def comap (f : α → β) (m : measurable_space β) : measurable_space
     let ⟨s', hs'⟩ := classical.axiom_of_choice hs in
     ⟨⋃ i, s' i, m.measurable_set_Union _ (λ i, (hs' i).left), by simp [hs'] ⟩ }
 
+lemma comap_eq_generate_from (m : measurable_space β) (f : α → β) :
+  m.comap f = generate_from {t | ∃ s, measurable_set s ∧ f ⁻¹' s = t} :=
+begin
+  have : {t | ∃ s, measurable_set s ∧ f ⁻¹' s = t} = (m.comap f).measurable_set' := rfl,
+  rw this,
+  convert generate_from_measurable_set.symm,
+end
```

Comment:
```suggestion
by { convert generate_from_measurable_set.symm }
```

Reply:
It feels like I tried that one several times, but I guess I didn't? Thanks.